Application/EdFi.Ods.Features/OpenApiMetadata/Strategies/FactoryStrategies/OpenApiMetadataDefinitionsFactoryDefaultEdFiExtensionBridgeStrategy.cs

```diff
@@ -11,44 +11,45 @@
 
 namespace EdFi.Ods.Features.OpenApiMetadata.Strategies.FactoryStrategies
```

Comment:
lets reformat this whole document please. its a little wonky

Reply:
Fixed